features/user_auth.feature

```diff
@@ -70,22 +70,22 @@ Feature: User Authentication with Devise
     And I should see Sign In
     And I should see Signed out successfully.
 
```

Comment:
The scenarios below shouldn't be commented out; did you rebase from Iter4Staging? Also, make sure that you're merging into Iter4Staging? You check that by clicking the "Edit" button at the top of the pull request